Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -716,7 +725,7 @@ private void QueryHistory()
 
         private readonly IReadOnlyList<Result> _emptyResult = new List<Result>();
 
-        private async void QueryResults()
+        private async void QueryResults(bool reQuery = false)
```

Comment:
Should we call these isReQuery so it's consistent with the query property?

Reply:
I used `reQuery` to match the previously named parameters in:
* `ChangeQueryText(string queryText, bool reQuery = false)` 
* `IPublicAPI.ChangeQuery(string query, bool requery = false)`

think I should change it?

Reply:
Ideally yes, but will this break any plugins that use this method?

Reply:
probably not during runtime but it could cause build issues. I can update all except the `IPublicApi` method...

Reply:
yeah let's do that.